tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py27,py34,py35,pypy
```

Comment:
We'll definitely drop support for the ancient Pythons in the next release. Makes sense.